model/patch/patch.go

```diff
@@ -48,7 +48,7 @@ type Patch struct {
 	VariantsTasks   []VariantTasks   `bson:"variants_tasks"`
 	Patches         []ModulePatch    `bson:"patches"`
 	Activated       bool             `bson:"activated"`
-	PatchedConfig   string           `bson:"patched_config"`
+	PatchedConfig   string           `bson:"patched_config"` // TODO: should we be adding support for patches
```

Comment:
I suspect not. Are these used to generate diffs in the UI? I don't think I've read this code before.